src/SB/Core/gc/iTRC.cpp

```diff
@@ -389,41 +388,44 @@ void iTRCDisk::SetMovieResumeFunction(void (*Func)())
 
 bool iTRCDisk::IsDiskIDed()
 {
-  S32 diskChk;
-  bool diskIDed;
-  
-  diskIDed = 0;
-  diskChk = DVDCheckDisk();
+    S32 diskChk;
+    bool diskIDed;
+
+    diskIDed = FALSE;
```

Comment:
This is kind of confusing so I totally understand why you used FALSE here, and it technically compiles but:

Use the C++ keyword `false` for working with C++ `bool` data types.
Use the C-style macro FALSE for working with C types (S32, S8, U8, etc.)

Same applies for TRUE/true usage.